frontera/contrib/backends/redis_backend/__init__.py

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8 -*-
+from collections import Iterable
+from datetime import datetime
+from frontera.utils.url import parse_domain_from_url_fast
+from frontera import DistributedBackend
+from frontera.core.components import Metadata, Queue, States
+from frontera.core.models import Request
+from frontera.contrib.backends.partitioners import Crc32NamePartitioner
+from frontera.utils.misc import get_crc32
+from frontera.contrib.backends.remote.codecs.msgpack import Decoder, Encoder
```

Comment:
Why not loading appropriate codecs in a way how it's done at `MessageBusBackend`?